interfaces/cython/cantera/onedim.py

```diff
@@ -532,10 +535,10 @@ def write_csv(self, filename, species='X', quiet=True):
         csvfile = open(filename, 'w')
         writer = _csv.writer(csvfile)
         writer.writerow(['z (m)', 'u (m/s)', 'V (1/s)', 'T (K)',
-                         'phi (V)', 'E (V/m)', 'rho (kg/m3)'] + self.gas.species_names)
+                         'phi (V)', 'E (V/m)', 'rho (kmol/m3)'] + self.gas.species_names)
         for n in range(self.flame.n_points):
             self.set_gas_state(n)
-            writer.writerow([z[n], u[n], V[n], T[n], phi[n], E[n], self.gas.density] +
+            writer.writerow([z[n], u[n], V[n], T[n], phi[n], E[n], self.gas.density_mole] +
```

Comment:
Is there a reason for this method output molar density rather than mass density? I'd rather mass density for consistency with what `FlameBase.write_csv` does (and has always done).